nomad/integration-tests-new.nomad

```diff
@@ -85,21 +96,23 @@ job "integration-tests-new" {
       connect {
         sidecar_service {
           proxy {
+            upstreams {
+              destination_name = "dgraph-alpha-0-grpc-public"
+              local_bind_port  = 9080
+            }
```

Comment:
i'd argue we could make this 1003 - the local bind port really shouldn't be important

Reply:
fixed